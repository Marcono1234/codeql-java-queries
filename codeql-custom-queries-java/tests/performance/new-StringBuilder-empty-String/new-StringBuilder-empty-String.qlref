./performance/new-StringBuilder-empty-String.ql
