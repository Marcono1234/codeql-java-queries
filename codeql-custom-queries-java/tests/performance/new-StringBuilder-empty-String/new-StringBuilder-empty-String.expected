| Test.java:5:9:5:29 | new StringBuilder(...) | Should create StringBuilder without arguments |
| Test.java:6:9:6:28 | new StringBuffer(...) | Should create StringBuffer without arguments |
